web/fanfare/src/config.ts

```diff
@@ -18,12 +20,23 @@ export default mergeSettings({
         //         key: process.env.GRAPHQL_TOKEN,
         //     },
         // ),
-        // TransactionObserver(
-        //     'wss://localhost:4000/rpc',
-        //     EthereumConnector,
-        //     {
-        //         key: process.env.ETHEREUM_TOKEN,
-        //     },
-        // ),
+        TransactionObserver({
+            connector: EthereumConnector({
+                chainRpcUrl: process.env.FLU_ETH_WS_RPC ?? '',
+                name: 'ethereum',
+                contractAddressSet: [
+                    "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
```

Comment:
Is this all Ethereum specific or does it apply to other EVM chains as well?

Reply:
Other chains too! Any ETH RPC compatible.

Reply:
Would be a lot better if this was supplied via environment variables the same way as the rest of the codebase (separated with ,)